Targets/Windows/SRUM.tkape

```diff
@@ -14,6 +14,26 @@ Targets:
         Category: Execution
         Path: C:\Windows.old\Windows\System32\SRU\
         Recursive: True
+    -
+        Name: SOFTWARE registry hive
+        Category: Registry
+        Path: C:\Windows\System32\config\
+        FileMask: SOFTWARE
+    -
+        Name: SOFTWARE registry hive
+        Category: Registry
+        Path: C:\Windows.old\Windows\System32\config\
+        FileMask: SOFTWARE   
+         
```

Comment:
Did you --tlist this? It will probably shoot an error here by not having the -

Reply:
rathbuna, yeah,  I did.  

I see the error that you are pointing out and will resubmit the pull request.  i have attached a few screen shots.  --tlist does not complain and the target produced the desired results.  weird.

thanks for checking and thanks for all your contributions to these projects. 

<img width="1497" alt="2020-12-31_08-22-59" src="https://user-images.githubusercontent.com/16808320/103414143-4e637e00-4b42-11eb-9273-fa4baef08460.png">
<img width="1496" alt="2020-12-31_08-21-42" src="https://user-images.githubusercontent.com/16808320/103414147-528f9b80-4b42-11eb-8e01-a4c75cc81be6.png">
<img width="1372" alt="2020-12-31_08-24-02" src="https://user-images.githubusercontent.com/16808320/103414152-54f1f580-4b42-11eb-848a-a42f938115da.png">

Reply:
> 
> 
> rathbuna, yeah, I did.
> 
> I see the error that you are pointing out and will resubmit the pull request. i have attached a few screen shots. --tlist does not complain and the target produced the desired results. weird.
> 
> thanks for checking and thanks for all your contributions to these projects.
> 
> <img alt="2020-12-31_08-22-59" width="1497" src="https://user-images.githubusercontent.com/16808320/103414143-4e637e00-4b42-11eb-9273-fa4baef08460.png">
> 
> <img alt="2020-12-31_08-21-42" width="1496" src="https://user-images.githubusercontent.com/16808320/103414147-528f9b80-4b42-11eb-8e01-a4c75cc81be6.png">
> 
> <img alt="2020-12-31_08-24-02" width="1372" src="https://user-images.githubusercontent.com/16808320/103414152-54f1f580-4b42-11eb-848a-a42f938115da.png">

Interesting! You shouldn't need to resubmit the PR. If you just do another commit to your fork with the added -, it should automatically include that change within this PR before we merge. That's typically how it's worked for me. 

@EricZimmerman do you have any insight on the missing - and if it'll be an issue/why it's not complaining about it?

Reply:
Oh crap,  two versions of the target on the machine.   sorry for being an idiot.